scripts/text_to_speech_node.py

```diff
@@ -169,12 +176,14 @@ def speak(self, sentence_msg):
         req.voice = self.voice
         req.emotion = self.emotion
         req.blocking_call = False
+        rospy.logdebug("Topic:\n{}".format(req))
```

Comment:
Why 'Topic: ...'?

Reply:
Ah, because this is the callback for a topic. Maybe rename the callback to be super clear like `speak_srv` below